concepticondata/conceptlists/Chen-2012-888.tsv

```diff
@@ -335,7 +335,7 @@ Chen-2012-888-333	333	马鞍	saddle	677	3065	SADDLE		588	用品工具
 Chen-2012-888-334	334	鞭子	whip	678	2630	WHIP		597	用品工具
 Chen-2012-888-335	335	伞	umbrella	678	3105	UMBRELLA		583	用品工具
 Chen-2012-888-336	336	斧子	axe	678	677	AXE		605	用品工具
-Chen-2012-888-337	337	柴刀	chopper	679				633	用品工具
+Chen-2012-888-337	337	柴刀	chopper	679	2772	KITCHEN KNIFE		633	用品工具
```

Comment:
The same gloss in [Huang-1992-1820-633](https://concepticon.clld.org/values/Huang-1992-1820-633) is mapped to CLEAVER. I'd propose to follow this mapping.

Reply:
Yes, good point.